rflx/pyrflx/typevalue.py

```diff
@@ -204,9 +219,10 @@ def expr(self) -> Variable:
         return Variable(self._value)
 
     @property
-    def binary(self) -> str:
+    def to_bitstring(self) -> Bitstring:
         self._raise_initialized()
-        return format(self._type.literals[self._value].value, f"0{self.size}b")
+        assert isinstance(self._type, Enumeration)
```

Comment:
`assert isinstance(self._type, Enumeration)` should not be needed here.

Reply:
Fixed by 9f39fdc